pkg/interface/groups/src/js/components/lib/contact-card.js

```diff
@@ -323,6 +323,7 @@ export class ContactCard extends Component {
         this.setState({awaiting: false});
         props.history.push(`/~groups${destination}`);
       });
+      api.contactView.delete(props.path);
```

Comment:
I'm surprised this fires given we're about to push to another view.

Reply:
The other call is asynchronous, but good point, I can remove the race condition.

Reply:
The hook does not do the same action, as the contactHookRemove API in the JS is sending a contact-action to the group owner, not a contact-hook-action to ourselves.

Reply:
But looking at this, I could make the view perform the action. Making this change.